filament/src/details/Engine.cpp

```diff
@@ -364,6 +365,8 @@ void FEngine::init() {
                 assert_invariant(false);
 #endif
                 break;
+            case StereoscopicType::NONE:
```

Comment:
Here's one more place to handle the default materials. Please do the same approach as did for skybox here before submitting.

Reply:
done